client/plots/violin.renderer.js

```diff
@@ -450,18 +449,19 @@ function getLegendGrps(self) {
 
 	const addUncomputableValues = term => {
 		if (term?.term.values) {
+			const items = []
 			for (const k in term.term.values) {
 				if (self.data.uncomputableValueObj?.[term.term.values[k]?.label]) {
-					const items = []
 					items.push({
 						text: `${term.term.values[k].label}, n = ${self.data.uncomputableValueObj[term.term.values[k].label]}`,
 						noIcon: true
 					})
-					title = `${term.term.name}`
```

Comment:
@karishma-gangwani This code was creating one legend group for each uncomputable value. I moved it outside the loop so that all of the uncomputable values will be under one group.